azext_iot/tests/central/test_iot_central_int.py

```diff
@@ -582,8 +665,8 @@ def _create_device_template(self):
         template_id = template_name + "id"
 
         self.cmd(
-            "iot central device-template create --app-id {} --device-template-id {} -k '{}'".format(
-                APP_ID, template_id, device_template_path
+            "iot central device-template create --app-id {} --device-template-id {} -k '{}' --token \"{}\"  --central-dns-suffix \"{}\" ".format(
+                APP_ID, template_id, device_template_path, TOKEN, DNS_SUFFIX
```

Comment:
@digimaun do we need to revert these DNS_SUFFIX/TOKEN params? Or should we update the IT pipeline to include them?

Reply:
I value ease of test configuration & maintaining tests -- if they are not strictly needed I would lean on removing it. Especially if they are not commonly used parameters by end users - it would be good to run command IT's in the most realistic way an end user would. We could have unit tests that cover DNS_SUFFIX & TOKEN plumbing. Open to more thoughts.

Reply:
These parameters have been introduced before and i fixed their consistency usage. domain prefix and token usage is valid for multi cloud configuration such as mooncake, fairfax, blackforest and allows to run integration tests in pre production enviroments

Reply:
But `--token` & `--central-dns-suffix` are not being applied consistently in `test_iot_central_int.py`. For example (arbitrarily choosing) 

https://github.com/Azure/azure-iot-cli-extension/blob/43b50797108368d0815ec2abfd157bb894188977/azext_iot/tests/central/test_iot_central_int.py#L93

https://github.com/Azure/azure-iot-cli-extension/blob/43b50797108368d0815ec2abfd157bb894188977/azext_iot/tests/central/test_iot_central_int.py#L336

https://github.com/Azure/azure-iot-cli-extension/blob/43b50797108368d0815ec2abfd157bb894188977/azext_iot/tests/central/test_iot_central_int.py#L597

Even if they were applied across the board, these parameters are optional. So we wouldn't be testing behavior without them which is probably common (not to include them).

There are several ways we could support this. We can do a whole pass of the central integration tests without them then a pass with them. This could be supported with a fixture addition. Another is to expand unit tests to assert the dns suffix and token plumbing.